lib/phonebook/providers/association_list_provider.dart

```diff
@@ -16,8 +16,7 @@ class AssociationListNotifier extends ListNotifier<Association> {
   }
 
   Future<AsyncValue<List<Association>>> loadAssociations() async {
-    associationList = await loadList(() async => associationRepository.getAssociationList());
-    return associationList;
+    return await loadList(associationRepository.getAssociationList);
```

Comment:
You are not loading this result in associationList here, is it supposed to be normal ?

Reply:
Fixed in next commit